x-pack/plugins/ml/server/routes/results_service.ts

```diff
@@ -265,4 +283,66 @@ export function resultsServiceRoutes({ router, mlLicense }: RouteInitialization)
       }
     })
   );
+
+  /**
+   * @apiGroup ResultsService
+   *
+   * @api {get} /api/ml/results/:jobId/categorizer_stats
+   * @apiName GetCategorizerStats
+   * @apiDescription Returns the categorizer snapshots for the specified job ID
+   * @apiSchema (params) jobIdSchema
+   * @apiSchema (query) getCategorizerStatsSchema
+   */
+  router.get(
+    {
+      path: '/api/ml/results/{jobId}/categorizer_stats',
+      validate: {
+        params: jobIdSchema,
+        query: getCategorizerStatsSchema,
+      },
+      options: {
+        tags: ['access:ml:canGetJobs'],
+      },
+    },
+    mlLicense.fullLicenseAPIGuard(async ({ legacyClient, request, response }) => {
+      try {
+        const resp = await getCategorizerStats(legacyClient, request.params, request.query);
+        return response.ok({
+          body: resp,
+        });
+      } catch (e) {
+        return response.customError(wrapError(e));
+      }
+    })
+  );
+
+  /**
+   * @apiGroup ResultsService
+   *
+   * @api {get} /api/ml/results/stopped_partitions
+   * @apiName GetStoppedPartitions
+   * @apiDescription Returns list of partitions we stopped categorizing whens status changed to warn
+   * @apiSchema (body) getStoppedPartitionsSchema
+   */
+  router.post(
+    {
+      path: '/api/ml/results/stopped_partitions',
```

Comment:
This endpoint is categorization job specific, so should probably be something like `/api/ml/results/category_stopped_partitions` for consistency with the existing endpoints.

Reply:
Updated here https://github.com/elastic/kibana/pull/75061/commits/7d8c39b0fc8d740f237a4fe9f3c4ef6644c5a4ec